code/modules/jobs/job_types/chief_engineer.dm

```diff
@@ -23,6 +23,7 @@
 	minimal_access = list(ACCESS_ENGINE, ACCESS_ENGINE_EQUIP, ACCESS_TECH_STORAGE, ACCESS_MAINT_TUNNELS,
 					ACCESS_EXTERNAL_AIRLOCKS, ACCESS_ATMOSPHERICS, ACCESS_EVA, ACCESS_AUX_BASE,
 					ACCESS_HEADS, ACCESS_CONSTRUCTION, ACCESS_SEC_DOORS, ACCESS_MINISAT, ACCESS_MECH_ENGINE,
+					ACCESS_WEAPONS, // Tegustation Heads of Staff eguns edit: Gives them a weapon's permit
```

Comment:
You actually could've done it in a better way.
Just make a new file for job edits and put something like this:
`minimal_access += ACCESS_WEAPONS`